cdist/conf/type/__install_config/files/remote/copy

```diff
@@ -37,10 +37,10 @@ code="$(echo "$@" | sed "s|$target_host:|$target_host:$chroot|g")"
 
 log "target_host: $target_host"
 log "chroot: $chroot"
-log "@: $@"
+log "@: $*"
 log "code: $code"
 
 # copy files into chroot
-$__default_remote_copy $code
+"$__default_remote_copy" "$code"
```

Comment:
Not 100% sure if this will break or not. But $code is supposed to be expanded. It is not a single string. It's n independent arguments. Same as passing $@ when exec'ing a command.

Reply:
I was unsure about this too. That is why I asked you to review it in detail.